### ISSUE TITLE
NVDA Ruler seems somewhat unresponsive and causes freeze

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-force-lab#256]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Firefox
**Problem description**
For https://github.com/phetsims/QA/issues/480
When using keyboard nav to move the ruler when using NVDA there are some responsiveness issues. When you first pick it up, there does not seem to be any indication of this with the border. Using the arrow keys will move the ruler, but often it does not do it fully/shows lag. Can be a bit inconsistent. 

**Visuals**
https://drive.google.com/file/d/1jTXHx7bi-OwJMbYdrZ_-m-HBKemIPUjo/view?usp=sharing
https://drive.google.com/file/d/1j1k4ZhH1Yqss_oFBi_sYsvcbXVY2Zhrf/view?usp=sharing

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-rc.2/phet/gravity-force-lab_all_phet.html
Version: 2.2.0-rc.2 2020-02-13 23:11:52 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:73.0) Gecko/20100101 Firefox/73.0
Language: en-US
Window: 1536x750
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by KatieWoe** [comment]
These console errors appear:
![consoleerrors](https://user-images.githubusercontent.com/41024075/74955912-9104e780-53c2-11ea-9928-9bf45ef98a9d.png)
If you stop NVDA then the sim freezes and you can't interact with anything after that.

**Comment by KatieWoe** [comment]
Happens in rc.1 and seems to need nvda to occur

**Comment by KatieWoe** [comment]
Does not seem to happen with Mac VO + Safari. I also noticed that part of the above problem is that the screen reader doesn't seem to read out as many alerts/information as it should.

**Comment by zepumph** [comment]
@KatieWoe can you please reproduce in the debug version of the sim and provide the console error for that?

**Comment by KatieWoe** [comment]
![moreerrors](https://user-images.githubusercontent.com/41024075/74970553-c9fd8600-53db-11ea-95da-15027d0a675e.png)

**Comment by KatieWoe** [comment]
![stillsame](https://user-images.githubusercontent.com/41024075/74972274-a5ef7400-53de-11ea-8310-fbd291a530d8.png)

**Comment by zepumph** [comment]
For some reason NVDA is sending touch events coming from the PDOM grab button when releasing the ruler. I told scenery to ignore it in the above commit. I also found that SceneryEventIO wasn't as tolerant as its core type was in serialization. above changes coupled with https://github.com/phetsims/axon/issues/287 make me feel confident that this is fixed and ready to move into a the RC branch.

**Comment by zepumph** [comment]
@KatieWoe thanks for the error logs, they helped in tracking things down.

**Comment by KatieWoe** [comment]
Looks good in rc3
